spec/controllers/game_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+RSpec.describe GameController do
+  def app
+    Rack::Builder.parse_file('config.ru').first
+  end
+
+  let(:code_out_of_range) { '8888' }
+  let(:long_code) { '11111' }
```

Comment:
It would be better to use here constant that you use in our validations here. For example:
`'1' * (Codebreaker::Validator::MAX_LENGTH + 1)`
Because if something changes in game rules something changes you will have to change all your tests

Reply:
+